src/apps/bm_soft_module/user_code/user_code.cpp

```diff
@@ -206,7 +206,7 @@ static bool BmSoftWatchdogHandler(void *arg) {
 
 static int createBmSoftDataTopic(void) {
   int topiclen = snprintf(bmSoftTopic, BM_TOPIC_MAX_LEN,
-                          "sensor/%" PRIx64 "/sofar/bm_soft_temp", getNodeId());
+                          "sensor/%016" PRIx64 "/sofar/bm_soft_temp", getNodeId());
```

Comment:
This PR mostly changes logs. However, this is one of the types of more load-bearing changes — the pub-sub sensor reading topics contain node IDs, and we want to make sure they match on both sides, always containing leading zeros if necessary to reach 16 hex digits.

This is topic on which the SOFT module publishes.